lib/modules/manager/github-actions/readme.md

```diff
@@ -40,7 +40,7 @@ name: build
 on: [push]
 
 env:
-  RUNNER: ubuntu-20.04
+  RUNNER: ubuntu-22.04
```

Comment:
Can we use `ubuntu-24.04` here? I was not sure, so to be safe I set it to the previous LTS (Jammy Jellyfish `22.04`) for now.

Let me know if this can be updated to the latest LTS! :wink:

Reply:
is in private beta

Reply:
Let's stick with 22.04 for now then.